### PR TITLE
Add dlib doc

### PR DESCRIPTION
**Pull request by pvizeli**
**Description:**

Add doc for dlib face platforms

### CODE REVIEW

File: source/_components/image_processing.dlib_face_detect.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description: "Instructions how to integrate Dlib Face Detect into Home Assistant."
+date: 2017-05-05 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: dlib.png
+ha_category: Image Processing
+featured: false
+ha_release: 0.44
```
Comment (Landrash) [inline source/_components/image_processing.dlib_face_detect.markdown]: 0.45

File: source/_components/image_processing.microsoft_face_detect.markdown [hunk 2]
```diff
@@ -31,7 +31,6 @@ image_processing:
 
 Configuration variables:
 
-- **group** (*Required*): Microsoft Face group used to detect the person.
```
Comment (Landrash) [inline source/_components/image_processing.microsoft_face_detect.markdown]: Is this change related to this component?

**Comment by arsaboo** [comment]
May be we should highlight that that it may not work on a Pi.
